### ISSUE TITLE
Diagram from deserialization using wrong Link model?

### ISSUE THREAD

**Issue by rzfzr** [projectstorm/react-diagrams#925]
You can see that my diagram uses "MyRightAngleLinkModel",
![image](https://user-images.githubusercontent.com/13456224/158497502-c0e34b0e-dcaf-4457-81f1-c90edfb4f071.png)
After serialization/deserialization, the existing links are right but the new ones are default... 
![image](https://user-images.githubusercontent.com/13456224/158497703-fc4927e9-606e-49c6-a13c-21f9332eba81.png)
I can notice that my portmodel also defaults back, as mine can connect out to out, after deserialization I can't anylonger

I tried deregistering the default factory with no results.... (this.diagramEngine.getLinkFactories().deregisterFactory('default'))

![image](https://user-images.githubusercontent.com/13456224/158497804-766606d9-d86f-4d42-bbdc-bc70e1edfd69.png)

**Comment by ShrutiChandak19** [comment]
HI @rzfzr  , did you found any solution for this?

**Comment by rzfzr** [comment]
@ShrutiChandak19 I tried figuring it out again and gave up... I bet it is something wrong with the deserialization process itself, not some options we can change.
I won't have a save/load function in my app.

**Comment by ShrutiChandak19** [comment]
@rzfzr yeah even I am thinking the same. There is some issue with deserialization process itself because after that only I'm getting issue.
